app/styles/components/_nypr-player-overrides.scss

```diff
@@ -0,0 +1,50 @@
+@import '../mixins/mq';
+
+$nypr-player-medium-breakpoint               : #{$mq_tablet-portrait};
+$nypr-player-medium-and-up                   : "(min-width: #{$nypr-player-medium-breakpoint})" !default;
+$nypr-player-small-only                      : "(max-width: #{$nypr-player-medium-breakpoint - 1})" !default;
+
+
+// text-crawl overflow styles
+.text-crawl {
```

Comment:
What's the reason for a `_nypr-player-overrides.scss` file at this point?

Reply:
w/o these the sticky player doesn't have text. It's just blank